google/README.txt

```diff
@@ -11,21 +11,18 @@ our development, so are not necessarily generalized or applicable outside GCE.
 General layout is this:
 
 google/
-   cassandra   Cassandra schemas, perhaps these could go to config.
-
-   config      Our *-local.yml overrides and default master configuration.
-               These are installed as "config_templates".
+   codelab     (Ignore) Supporting script for building codelab image
 
    dev         Scripts meant for development support, not operational usage.
                These are not included in a typical deployment, though are
                still open source for developers.
 
-   install     These are release installation scripts. They are kept separate
-               from the runtime admin scripts for convenience.
+   google_cloud_logging
+               (Optional) Script to install Google Cloud Logging and
+               Monitoring integration.
 
-   pylib       A python package containing modules used by the runtime scripts
-               All these are part of the runtime environment. The developer
-               scripts and tools put their python in the dev directory.
+   stackdriver_monitoring
+               (Optional) Configurations to enable StackDriver integration.
```

Comment:
That was the original intent, but monitoring is baked into spinnaker itself. Instead I have supporting tools in here. The tools are development oriented for the time being. Maybe this is "administrative support for Spinnaker Stackdriver integration".